lib/commands/ci.js

```diff
@@ -3,12 +3,16 @@ const runScript = require('@npmcli/run-script')
 const fs = require('fs/promises')
 const { log, time } = require('proc-log')
 const validateLockfile = require('../utils/validate-lockfile.js')
-const ArboristWorkspaceCmd = require('../arborist-cmd.js')
+const ArboristCmd = require('../arborist-cmd.js')
```

Comment:
I think this needs to be an Arborist workspace command. The "clean" part needs to remove all node_modules, but the reify/install still can get workspace flags added to it.